build.sbt

```diff
@@ -130,19 +128,49 @@ lazy val commonSettings = Seq(
 // General Settings
 inThisBuild(List(
   organization := "org.typelevel",
-  organizationName := "Typelevel Contributors",
-  developers := List(
-    Developer("rossabaker", "Ross A. Baker", "ross@rossabaker.com", url("https://github.com/rossabaker"))
-  ),
+  organizationName := "Typelevel",
+  publishGithubUser := "rossabaker",
+  publishFullName := "Ross A. Baker",
+  baseVersion := "0.3",
 
   homepage := Some(url("https://github.com/typelevel/case-insensitive")),
   startYear := Some(2020),
   licenses := Seq("Apache-2.0" -> url("https://www.apache.org/licenses/LICENSE-2.0.html")),
+  scmInfo := Some(ScmInfo(url("https://github.com/typelevel/case-insensitive"),
+    "git@github.com:typelevel/case-insensitive.git")),
 
-  pomIncludeRepository := { _ => false},
-  scalacOptions in (Compile, doc) ++= Seq(
-      "-groups",
-      "-sourcepath", (baseDirectory in LocalRootProject).value.getAbsolutePath,
-      "-doc-source-url", "https://github.com/typelevel/case-insensitive/blob/v" + version.value + "€{FILE_PATH}.scala"
+  githubWorkflowTargetTags ++= Seq("v*"),
+  githubWorkflowPublishTargetBranches := Seq(RefPredicate.StartsWith(Ref.Tag("v"))),
+  githubWorkflowBuildPreamble := Seq(
+    WorkflowStep.Use("olafurpg", "setup-gpg", "v3"),
+    WorkflowStep.Use("actions", "setup-ruby", "v1"),
+    WorkflowStep.Run(List("""
+      |gem install bundler
+      |bundle install --gemfile=site/Gemfile
+    """.stripMargin), name = Some("Install Jekyll"))
+  ),
+  githubWorkflowBuild := Seq(
+    WorkflowStep.Sbt(List(
+      "headerCheck",
+      "test:headerCheck",
+      "scalafmtCheckAll",
+      "testIfRelevant",
+      "mimaReportBinaryIssuesIfRelevant",
+      "doc",
+      "makeMicrosite"
```

Comment:
I'm not feeling good about these running on Dotty.  Are we going to need `IfRelevant` tasks?

Reply:
More than likely, yes :-( Fortunately it's a two-liner:

```scala
lazy val makeMicrositeIfRelevant = taskKey[Unit]("documentation and stuff")
ThisBuild / makeMicrositeIfRelevant := filterTaskWhereRelevant(makeMicrosite)
```

Reply:
I'd like to upstream that into sbt-spiewak, but that's another dependency for that.